### PR TITLE
Polish Kubernetes doc

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: docs/src/main/asciidoc/kubernetes.adoc [hunk 1]
```diff
@@ -527,14 +521,15 @@ To enable the generation of OpenShift resources, you need to include OpenShift i
 quarkus.kubernetes.deployment-target=openshift
 ----
 
-If you need to generate resources for both platforms (vanilla Kubernetes and OpenShift), then you need to include both (coma separated).
+If you need to generate resources for both platforms (vanilla Kubernetes and OpenShift), then you need to include both (comma separated).
```
Comment (machi1990) [inline docs/src/main/asciidoc/kubernetes.adoc]: good catch!

File: docs/src/main/asciidoc/kubernetes.adoc [hunk 2]
```diff
@@ -8,7 +8,7 @@ https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
 include::./attributes.adoc[]
 
 This guide covers generating and deploying Kubernetes resources based on sane defaults and user supplied configuration.
-In detail, it supports generating resources for <<Kubernetes>>, <<OpenShift>> and <<Knative>>. Also it supports automatically <<Deployment>> these resources to the target platform.
+In detail, it supports generating resources for <<Kubernetes>>, <<OpenShift>> and <<Knative>>. Also it supports automatic deployment of these resources to the target platform.
```
Comment (gsmet) [inline docs/src/main/asciidoc/kubernetes.adoc]: I prefer we avoid having links without ids. If we change the titles, they get broken.

File: docs/src/main/asciidoc/kubernetes.adoc [hunk 3]
```diff
@@ -248,7 +246,7 @@ That is because for environment variables there are additional options rather th
 
 ===== Environment variables from Secret
 
-To add all key value pairs of a `Secret` as environment variables:
+To add all key value pairs of a `Secret` as environment variables just apply the following configuration:
```
Comment (gsmet) [inline docs/src/main/asciidoc/kubernetes.adoc]:
```suggestion
To add all key/value pairs of a `Secret` as environment variables just apply the following configuration:
```
while you're at it :)

File: docs/src/main/asciidoc/kubernetes.adoc [hunk 4]
```diff
@@ -257,7 +255,7 @@ quarkus.kubernetes.env-vars.my-env-var.secret=my-secret
 
 ===== Environment variables from ConfigMap
 
-To add all key value pairs of a `ConfigMap` as environment variables:
+To add all key value pairs of a `ConfigMap` as environment variables just apply the following configuration:
```
Comment (gsmet) [inline docs/src/main/asciidoc/kubernetes.adoc]:
```suggestion
To add all key/value pairs of a `ConfigMap` as environment variables just apply the following configuration:
```

File: docs/src/main/asciidoc/kubernetes.adoc [hunk 5]
```diff
@@ -527,14 +521,15 @@ To enable the generation of OpenShift resources, you need to include OpenShift i
...
-If you need to generate resources for both platforms (vanilla Kubernetes and OpenShift), then you need to include both (coma separated).
+If you need to generate resources for both platforms (vanilla Kubernetes and OpenShift), then you need to include both (comma separated).
 
 [source]
 ----
 quarkus.kubernetes.deployment-target=kubernetes, openshift
 ----
 
-Note: In latest versions of Quarkus a wrapper extension of link:openshift[Openshift] is provided. If you use it, there is no need to explicitly set this property.
+NOTE: Quarkus also provides the link:openshift[Openshift] extension. This extension is basically wrapper around the Kubernetes extension and
```
Comment (gsmet) [inline docs/src/main/asciidoc/kubernetes.adoc]:
```suggestion
NOTE: Quarkus also provides the link:openshift[OpenShift] extension. This extension is basically wrapper around the Kubernetes extension and
```

File: docs/src/main/asciidoc/kubernetes.adoc [hunk 6]
```diff
@@ -527,14 +521,15 @@ To enable the generation of OpenShift resources, you need to include OpenShift i
...
+relieves Openshift users of the necessity of setting the `deployment-target` property to `openshift`
```
Comment (gsmet) [inline docs/src/main/asciidoc/kubernetes.adoc]:
```suggestion
relieves OpenShift users of the necessity of setting the `deployment-target` property to `openshift`
```

Comment (gsmet) [inline docs/src/main/asciidoc/kubernetes.adoc]: Could you maybe check the rest of the doc to be sure the case is right?

File: docs/src/main/asciidoc/kubernetes.adoc [hunk 7]
```diff
@@ -674,7 +669,7 @@ The properties for configuring `docker` and `s2i` are also deprecated in favor o
 
 ==== Config group arrays
 
-Properties refering to config group arrays (e.g. kubernetes.labels[0], kubernetes.env-vars[0] etc) have been converted to maps, to align with the rest of the quarkus ecosystem.
+Properties referring to config group arrays (e.g. kubernetes.labels[0], kubernetes.env-vars[0] etc) have been converted to maps, to align with the rest of the quarkus ecosystem.
```
Comment (gsmet) [inline docs/src/main/asciidoc/kubernetes.adoc]:
```suggestion
Properties referring to config group arrays (e.g. `kubernetes.labels[0]`, `kubernetes.env-vars[0]` etc) have been converted to maps, to align with the rest of the Quarkus ecosystem.
```

Comment (gsmet) [inline docs/src/main/asciidoc/kubernetes.adoc]: Same here, can you check the case of `Quarkus` throughout the doc?

**Review by gsmet — CHANGES_REQUESTED** [review]
Thanks for taking the time to proofread that. I added a couple suggestions.

**Comment by geoand** [comment]
Thanks for checking this folks! I'll probably address your comments tomorrow

**Comment by machi1990** [comment]
@geoand Just noticed that we do not use the generated doc in this guide e.g here https://github.com/quarkusio/quarkus/blob/58127197e43b08bb251184d57f53d1515c17f94a/docs/src/main/asciidoc/kubernetes.adoc#configuration-options is there any reason?

**Comment by geoand** [comment]
I wasn't sure what it would look like and what extra information @iocanel has added to the manually added tables. Obviously we want to use the generated ones if possible

**Comment by machi1990** [comment]
> I wasn't sure what it would look like and what extra information @iocanel has added to the manually added tables. Obviously we want to use the generated ones if possible

Yes, if possible let's use it. WDYT? @iocanel

File: docs/src/main/asciidoc/kubernetes.adoc (hunk 2 above)
Comment (geoand) [reply docs/src/main/asciidoc/kubernetes.adoc]: I can't say I know much about adoc, but I think I fixed it now?

File: docs/src/main/asciidoc/kubernetes.adoc (hunk 6 above)
Comment (geoand) [reply docs/src/main/asciidoc/kubernetes.adoc]: Done!

File: docs/src/main/asciidoc/kubernetes.adoc (hunk 7 above)
Comment (geoand) [reply docs/src/main/asciidoc/kubernetes.adoc]: Yup, fixes

File: docs/src/main/asciidoc/kubernetes.adoc (hunk 2 above)
Comment (gsmet) [reply docs/src/main/asciidoc/kubernetes.adoc]: The whole point of it is to define ids on the paragraphs so that you refer to fixed ids

Comment (geoand) [reply docs/src/main/asciidoc/kubernetes.adoc]: Gotcha, thanks (as you already know, I never developed a taste for docs 😂). I'll fix it tomorrow morning

Comment (geoand) [reply docs/src/main/asciidoc/kubernetes.adoc]: Was able to fix it quickly. I assume it's correct now?

**Review by gsmet — APPROVED** [review]

**Pull request merged**